### ISSUE TITLE
Patch for /trunk/javascriptguide.xml

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/google-styleguide#3]
```
The bad example in "Passing Anonymous Functions" was marked as good code.
```

---

Original issue reported on code.google.com by `max99x` on 24 Mar 2011 at 5:50

Attachments:
- [javascriptguide.xml.patch](https://storage.googleapis.com/google-code-attachments/google-styleguide/issue-3/comment-0/javascriptguide.xml.patch)

**Comment by GoogleCodeExporter** [comment]
```
This is fixed in the JavaScript Style Guide 2.20 release at r67.
```

---

Original comment by `mark@chromium.org` on 4 Nov 2011 at 6:09
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
---

Original comment by `mark@chromium.org` on 4 Nov 2011 at 9:18
